### ISSUE TITLE
Ability to forward a message

### ISSUE THREAD

**Issue by zlshames** [BlueBubblesApp/BlueBubbles-Android-App#454]

**Comment by zlshames** [comment]
Waiting on brandon to push his changes to the details popup widget

**Comment by zlshames** [comment]
This is kinda related to: https://github.com/BlueBubblesApp/BlueBubbles-Android-App/issues/885
We probably want to add a menu somewhere, not part of the context menu so that we can add as many options as we need to the details popup
